docs/howto/how-to-consume-metrics.md

```diff
@@ -0,0 +1,254 @@
+# Consuming Metrics
```

Comment:
Something we haven't really talked about or established a community-wide standard around: If something is not part of our public API, should we document it? If so, how and where?

As soon as something is documented, it is not unreasonable for consumers to believe it is part of the public API, so we need to be extremely careful about documenting things that fall outside that.

Metrics are not included in our public API. So the above question is very relevant to this PR.

Do we have any prior art here? @michaelryanpeter , what is your stance on how to handle this?

Reply:
During the review about remove the manufests, it was mentioned that we need to create documentation. To address this, I’ve prepared a draft for discussion before we invest more time into it.

The idea is to align on expectations—what should or shouldn’t be included—before proceeding further.

Here’s what we need to clarify:

1. Should we include details on how to consume the metrics generally? 
2. Or should we focus solely on consuming metrics via Prometheus?

If we choose the option 2, it might result in incomplete or uneven information.

Let’s discuss and agree on the direction before diving deeper into this.

Looking forward to your thoughts!

c/c @tmshort @tylerslaton @michaelryanpeter